testkit/src/main/scala/org/bitcoins/testkit/core/gen/ScriptGenerators.scala

```diff
@@ -579,10 +590,14 @@ sealed abstract class ScriptGenerators extends BitcoinSLogger {
 
     signed.flatMap {
       case (scriptSig, spk, keys) =>
-        rawScriptPubKey(defaultMaxDepth).map(_._1).map { spk2 =>
-          (ConditionalScriptSignature(scriptSig, true),
-           ConditionalScriptPubKey(spk.asInstanceOf[RawScriptPubKey], spk2),
-           keys)
+        conditionalOperation.flatMap { op =>
+          rawScriptPubKey(defaultMaxDepth).map(_._1).map { spk2 =>
+            (ConditionalScriptSignature(scriptSig, true),
+             ConditionalScriptPubKey(op,
+                                     spk.asInstanceOf[RawScriptPubKey],
```

Comment:
is this cast avoidable by using a diff generator?

Reply:
Added a type parameter to `packageToSequenceOfPrivateKeys` which was the cause for this cast